draft-sharabayko-mops-srt.md

```diff
@@ -3003,6 +2999,63 @@ IV = (MSB(112, Salt) << 2) XOR (PktSeqNo)
+
+The default SRT data transmission mode for continuous live streaming is message mode ({{transmission-mode-msg}}) with certain settings applied and described below:
+
+- Only data packets with their PP field set to "11b" are allowed, meaning a single data packet forms exactly one message ({{data-pkt}}).
+
+- Timestamp-Based Packet Delivery (TSBPD) ({{tsbpd}}) and Too-Late Packet Drop
+  (TLPKTDROP) ({{too-late-packet-drop}}) mechanisms must be enabled.
+
+- Live congestion control (LiveCC) ({{liveCC}}) must be used.
+
+- The Order Flag ({{data-pkt}}) needs special attention. In the case
+  of live streaming, it is set to 0 allowing out of order
+  delivery of a packet. However, in this use case the Order Flag has to be ignored
+  by the receiver. As TSBPD is enabled, the receiver will still deliver
+  packets in order, but based on the timestamps. In the case of a
+  packet arriving too late and skipped by TLPKTDROP mechanism, the
+  order of delivery is still maintained except for potential sequence
+  discontinuity.
+
+  This method has grown historically and is the current common standard
+  for live streaming across different SRT implementations. A change or
+  variation of the settings will break compatibility between two parties.
+
+This combination of settings allows live streaming with a constant latency ({{srt-latency}}). The receiving end will not "fall behind" in time by waiting for missing packets. However, the data integrity might not be ensured, if packets or retransmitted packets do not arrive within the expected time frame. Audio or video interruption can occur, but the overall latency is maintained and does not grow up over time whenever packets are missing.
+
+## File Transmission Use Case {#file-transmission-use-case}
+
+This section describes the use case of file transmission as well as provides configuration examples.
+
+The usage of both message and buffer modes ({{data-transmission-modes}}) is possible in this case. For both modes, Timestamp-Based Packet Delivery (TSBPD) ({{tsbpd}}) and Too-Late Packet Drop (TLPKTDROP) ({{too-late-packet-drop}}) mechanisms must be turned off while congestion control should be set to file transfer congestion control (FileCC) ({{fileCC}).
+
+When TSBPD is disabled, each packet gets the timestamp at the time it is sent by the SRT sender. A packet being sent for the first time will have different timestamp than a corresponding retransmitted packet. In contrast to the live streaming case, the timing of packets' delivery, when sending files, is not critical. The most important thing is data integrity. Therefore TLPKTDROP mechanism must be disabled in this case. No data is allowed to be dropped, because this will result in corrupted files with missing data. The retransmission of missing packets has to happen until the packets are finally acknowledged by the SRT receiver.
```

Comment:
```suggestion
When TSBPD is disabled, each packet gets timestamped with the time it is sent by the SRT sender. A packet being sent for the first time will have a timestamp different from that of a corresponding retransmitted packet. In contrast to the live streaming case, the timing of packets' delivery, when sending files, is not critical. The most important thing is data integrity. Therefore the TLPKTDROP mechanism must be disabled in this case. No data is allowed to be dropped, because this will result in corrupted files with missing data. The retransmission of missing packets has to happen until the packets are finally acknowledged by the SRT receiver.
```